polylux/recipe.yaml

```diff
@@ -0,0 +1,44 @@
+context:
+  name: polylux
+  version: "0.3.1"
+
+package:
+  name: typst-${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/andreasKroepelin/polylux/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: ad146e04657b3514b35873b45f4a7c33d582fe795e634f8aa4ac14b2074e0f3d
+
+build:
+  number: 2
+  noarch: generic
+  script:
+    content:
+      - mkdir -p "$PREFIX/share/typst/packages/typst-forge/${{ name }}/${{ version }}"
+      - cp -r typst.toml README.md polylux.typ logic.typ utils themes
+          "$PREFIX/share/typst/packages/typst-forge/${{ name }}/${{ version }}"
+
+tests:
+  - script:
+      - typst compile test.typ
+    files:
+      recipe:
+        test.typ
+
+requirements:
+  build:
+    - __unix
+  run:
+    - typst >=0.12.0rc1
```

Comment:
Can you change this to 0.12.0? I also want to rebuild all existing packages to not depend on rc.